projects/sandbox/src/app/dataset-info/dataset-info.component.html

```diff
@@ -29,7 +29,7 @@
       }"
       [attr.title]="canRunDebias ? 'run debias report' : 'view debias report'"
       (click)="runOrShowDebiasReport(canRunDebias, openerDebias)"
-      (keydown.enter)="runOrShowDebiasReport(canRunDebias, openerDebias)"
+      (keydown.enter)="runOrShowDebiasReport(canRunDebias, openerDebias, true)"
```

Comment:
from my suggestion: boolean parameters are not a good practice. they are a code smell. usually means a function needs to be split into two different functions.